server/src/protocol/server_protocol.rs

```diff
@@ -34,21 +34,39 @@ pub struct Vehicle {
     pub position: Position,
 }
 
-/// Represent a list of vehicles.
+/// Represent a line.
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Line {
     pub timestamp: String,
     pub line: String,
-    pub vehicles: i32,
+    pub vehicles: u32,
```

Comment:
We'll see if it is relevant to keep track of the number of buses later on, but we can come back to this later when we've implemented this.